src/engine/ruleset/decoders/fortinet/client-endpoint.yml

```diff
@@ -0,0 +1,56 @@
+---
+name: decoder/fortinet-client-endpoint/0
+
+metadata:
+  title: Decoder for Fortinet FortiClient Endpoint Security
+  description: Decoder for Fortinet FortiClient Endpoint Security
+  compatibility: >
+    Wazuh Engine
+  author:
+    name: Wazuh, Inc.
+    date: 2023/01/09
+  references:
+    - https://docs.fortinet.com/document/forticlient/7.0.2/log-message-reference/327109
+
+sources:
+  - decoder/queue-localfile/0
+
+parse:
+  logpar:
+    # January 29 06:09:59 boNemoe4402.www.invalid proto=udp service=http status=deny src=10.150.92.220 dst=10.102.123.34 src_port=7178 dst_port=3994 server_app=reeufugi pid=7880 app_name=enderitq traff_direct=external block_count=5286 logon_user=sumdo@litesse6379.api.domain msg=failure
```

Comment:
Improve-desing: you could capture this first part : 
`January 29 06:09:59 boNemoe4402.www.invalid`
and then use KV parser on the rest of the message. What do you think?

Reply:
Agree with the previous comment.

Reply:
done!